ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/ethernet-interfaces.cfg

```diff
@@ -318,6 +318,17 @@ interface Ethernet31
    switchport
    dot1x port-control force-unauthorized
 !
+interface Ethernet32
+   description DOT1X Testing - auto reauthentication
+   switchport
+   dot1x reauthentication
+   dot1x port-control auto
+!
+interface Ethernet33
```

Comment:
you don't need this test case. This case is basically covered by _all_ other interfaces in AVD tests :)